### PR TITLE
Undefined og description

### PR DESCRIPTION
**Pull request by billbitt**
updated controller to pass the required info (title and description) to the view renderer

### CODE REVIEW

File: controllers/publishController.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ module.exports = {
         resolve(publishResults); // resolve the promise with the result from lbryApi.publishClaim;
       })
       .catch(error => {
-        logger.error('publishController.publish, error', error);
+        logger.error('caught a publishController.publish error');
```
Comment (kauffj) [inline controllers/publishController.js]: Shouldn't the actual error be logged here?

If not, ignore.

File: helpers/handlebarsHelpers.js [hunk 2]
```diff
@@ -17,56 +17,48 @@ module.exports = {
...
-    let basicTags = `<meta property="og:title" content="${title}"> 
-          <meta property="og:url" content="${showUrl}" > 
-          <meta property="og:site_name" content="Spee.ch" > 
-          <meta property="og:description" content="${description}">`;
+    console.log('title', title);
+    console.log('description', description);
+    let ogTitle = `<meta property="og:title" content="${title}" >`;
+    let ogUrl = `<meta property="og:url" content="${showUrl}" >`;
+    let ogSiteName = `<meta property="og:site_name" content="Spee.ch" >`;
+    let ogDescription = `<meta property="og:description" content="${description}" >`;
```
Comment (kauffj) [inline helpers/handlebarsHelpers.js]: Is there a fallback value for this when it is not set?

File: controllers/publishController.js (hunk 1 above)
Comment (billbitt) [reply controllers/publishController.js]: The full error is passed back up the promise chain and then logged.  This particular log is more just for debugging, to be able to quickly ID if this error was triggered.

File: helpers/handlebarsHelpers.js (hunk 2 above)
Comment (billbitt) [reply helpers/handlebarsHelpers.js]: no there isn't, and you are correct that should be added :)

**Pull request merged**